parser_library/src/lsp/item_convertors.cpp

```diff
@@ -107,11 +108,11 @@ std::string hover_text(const context::symbol& sym)
         markdown.append("  \n");
     }
     if (attrs.is_defined(context::data_attr_kind::I))
-        markdown.append("I: " + std::to_string(attrs.integer()) + "  \n");
+        markdown.append("I: ").append(std::to_string(attrs.integer())).append("  \n");
     if (attrs.is_defined(context::data_attr_kind::S))
-        markdown.append("S: " + std::to_string(attrs.scale()) + "  \n");
+        markdown.append("S: ").append(std::to_string(attrs.scale())).append("  \n");
     if (attrs.is_defined(context::data_attr_kind::T))
-        markdown.append("T: " + ebcdic_encoding::to_ascii((unsigned char)attrs.type()) + "  \n");
+        markdown.append("T: ").append(ebcdic_encoding::to_ascii((unsigned char)attrs.type())).append("  \n");
```

Comment:
`std::format` cannot be used here?

Reply:
I though about it, but this is building a larger document, there are conditions etc. so I don't think it would really bring any value at this point.